releases/cert-manager/values-issuers-raw.yaml.gotmpl

```diff
@@ -0,0 +1,44 @@
+{{/* vim: set filetype=mustache: */}}
+{{- $issuers := tpl (readFile "templates/issuers.tpl") $ | fromYaml | get "issuers" list -}}
+templates:
+{{- if has "letsencrypt" $issuers }}
+  - |
+    ---
+    apiVersion: cert-manager.io/v1
+    kind: ClusterIssuer
+    metadata:
+      name: letsencrypt-prod
+      namespace: cert-manager
+    spec:
+      acme:
+        server: https://acme-v02.api.letsencrypt.org/directory
+        email: {{ .Values._set.letsencrypt.acme_email }}
+        privateKeySecretRef:
+          name: letsencrypt-prod
+        solvers:
+          # An empty 'selector' means that this solver matches all domains
+          - selector: {}
+            http01:
+              ingress:
+                class: nginx
```

Comment:
WDYT on making this value configurable?

Reply:
it might be likely separate syntax for say it DNS issuer, so configurable but kinda interchangable. But when we get there.

Reply:
I mean `class: nginx`. Having multiple ingresses w/ different classes is a common use-case, class hardcode may be not a best option in the future.

Reply:
yup this too